attribute-service-impl/src/main/java/org/hypertrace/core/attribute/service/constants/AttributeFieldPathConstants.java

```diff
@@ -0,0 +1,19 @@
+package org.hypertrace.core.attribute.service.constants;
+
+import org.hypertrace.core.attribute.service.v1.AttributeSource;
+
+public class AttributeFieldPathConstants {
+  public static final String ID_PATH = "id";
+  public static final String FQN_PATH = "fqn";
+  public static final String SCOPE_PATH = "scope";
+  public static final String SCOPE_STRING_PATH = "scope_string";
+  public static final String KEY_PATH = "key";
+  public static final String INTERNAL_PATH = "internal";
+
+  public static final String TENANT_ID_PATH = "tenant_id";
+  public static final String SOURCE_METADATA_PATH = "metadata";
+
+  public static String sourceMetadataPathFor(final AttributeSource source) {
```

Comment:
nit - not a constant, by definition 😉  - if it's only used in one spot, would leave it there